web/src/main/java/org/hillview/dataStructures/IntervalDecomposition.java

```diff
@@ -155,8 +157,8 @@ public int getQuantizationIntervalCount() {
     @SuppressWarnings("SameParameterValue")
     List<Pair<Integer, Integer>> bucketDecomposition(int bucketIdx, boolean cdf) {
         Pair<Integer, Integer> range = this.bucketRange(bucketIdx, cdf);
-        return IntervalDecomposition.dyadicDecomposition(
-            Converters.checkNull(range.first), Converters.checkNull(range.second));
+        return IntervalDecomposition.kadicDecomposition(
```

Comment:
Always?
No policy to decide?
We should add this to the JSON file as well. Would you like to do it in this PR?

Reply:
It can be a policy parameter, but I don't think adding this to the schema needs to be a priority right now; I can open an issue for it and we can address this later.